server/database/migrations/20190731135151-create-session.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Sessions', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      allowNull: false,
+      type: Sequelize.INTEGER
+    },
+    active: {
+      allowNull: false,
+      type: Sequelize.BOOLEAN,
+      defaultValue: true
+    },
+    devicePlatform: {
+      allowNull: false,
+      type: Sequelize.STRING,
+      defaultValue: 'browser'
+    },
+    expiresAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    ipAddress: {
+      allowNull: false,
```

Comment:
allow null here

Reply:
Done

Reply:
I think allow null here means that the field should be nullable, or am I wrong?